kubelet/remote/remote_image.go

```diff
@@ -29,14 +29,14 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/util"
 )
 
-// RemoteImageService is a gRPC implementation of internalapi.ImageManagerService.
-type RemoteImageService struct {
+// ImageService is a gRPC implementation of internalapi.ImageManagerService.
+type ImageService struct {
 	timeout     time.Duration
 	imageClient runtimeapi.ImageServiceClient
 }
 
-// NewRemoteImageService creates a new internalapi.ImageManagerService.
-func NewRemoteImageService(endpoint string, connectionTimeout time.Duration) (internalapi.ImageManagerService, error) {
+// NewImageService creates a new internalapi.ImageManagerService.
```

Comment:
change `NewRemoteImageService` to `NewImageService` to pass the static check?
I think it would be terrible when synchronizing with upstream.

Reply:
Yes, it will be better if we may write some rules to filter/ignore naming check.